source/mentoring-service/source/community-gathering.md

```diff
@@ -12,4 +12,5 @@
 
 ## USA/Seattle April 2023
 
-![USA ](/source/images/mics/usa-apr-23.jpg)
\ No newline at end of file
+![USA ](/source/images/mics/usa-apr-23.jpg)
```

Comment:
@naja-ahmed there is no change here. So, why you include this file as part of your changes?
Please remove this file from your changes.

Reply:
@naja-ahmed no you didn't, it's still in your PR.
Please try to undo this change so it doesn't show the file change in your PR

Reply:
I deleted the file and then commit changes , is there any thing else i have to do and i forget ?

Reply:
@naja-ahmed yes, you need to reset or undo the changes, can't you remember that in the session? You may need to review that session.